docs/items/index.md

```diff
@@ -1,72 +1,127 @@
+- `durability` - Sets the durability of this item. Defaults to 0, which means "no durability". For example, iron tools use 250 here. Note that setting the durability automatically locks the stack size to 1.
+- `craftRemainder` - Sets the crafting remainder of this item. Vanilla uses this for filled buckets that leave behind empty buckets after crafting.
+- `fireResistant` - Makes item entities that use this item immune to fire and lava. Used by various netherite items.
+- `setNoRepair` - Disables anvil and crafting grid repairing for this item. Unused in vanilla.
+- `rarity` - Sets the rarity of this item. Currently, this simply changes the item's color. `Rarity` is an enum consisting of the four values `COMMON` (white, default), `UNCOMMON` (yellow), `RARE` (aqua) and `EPIC` (light purple). Be aware that mods may add more rarity types.
+- `requiredFeatures` - Sets the required feature flags for this item. This is mainly used for vanilla's feature locking system in minor versions. It is discouraged to use this, unless you're integrating with a system locked behind feature flags by vanilla.
+- `food` - Sets the [`FoodProperties`][food] of this item.
+
+For examples, or to look at the various values used by Minecraft, have a look at the `Items` class.
+
+### Food
+
+The `Item` class provides default functionality for food items, meaning you don't need a separate class for that. To make your item edible, all you need to do is set the `FoodProperties` on it through the `food` method in `Item.Properties`.
+
+`FoodProperties` are created using a `FoodProperties.Builder`. You can then set various properties on it:
+
+- `nutrition` - Probably the most obvious part. Sets how many hunger points are restored. Counts in half hunger points, so for example, Minecraft's steak restores 8 hunger points.
+- `saturationMod` - The saturation modifier used in calculating the [saturation value][hunger] restored when eating this food. The calculation is `min(2 * nutrition * saturationMod, playerNutrition)`, meaning that using `0.5` will make the effective saturation value the same as the nutrition value.
+- `meat` - Whether this item should be considered meat or not. Used e.g. for determining if healing dogs with this food is possible.
+- `alwaysEat` - Whether this item can always be eaten, even if the hunger bar is full. `false` by default, `true` for golden apples and other items that provide bonuses beyond just filling the hunger bar.
+- `fast` - Whether fast eating should be enabled for this food. `false` by default, `true` for dried kelp in vanilla.
+- `effect` - Adds a `MobEffectInstance` to apply when eating this item. The second parameter denotes the probability of the effect being applied; for example, Rotten Flesh has an 80% chance of applying the Hunger effect when eaten. This method comes in two variants, one that takes in a supplier (for your own effects) and one that directly takes a `MobEffectInstance` (for vanilla effects).
```

Comment:
Do we still allow registry replacement? If not, ignore this comment.

Reply:
I actually don't know? But regardless, what exactly is this comment aiming at?